devops/scripts/ci-tester.sh

```diff
@@ -31,7 +31,7 @@ if [ "$?" == "0" ]; then
     esac
 fi
 
-if [ -z "$TEST_REPORTS" ]; then
+if [ -z "$TEST_REPORTS" ] || [ "$TEST_REPORTS" == '${CIRCLE_TEST_REPORTS}' ]; then 
```

Comment:
> Also, you have ${CIRCLE_TEST_REPORTS} wrapped in single, not double, quotes, so it will be treated as a literal string and variable expansion will not happen.

Yeah... that is intentional. There is a bug in CircleCI 2.0 where all of a sudden the env vars i defined from the web UI arent getting expanded. I'm trying to catch that scenario and fix it without breaking currently passing functionality in CircleCI 1.0.